include/asterisk/pbx.h

```diff
@@ -417,6 +417,26 @@ struct ast_pbx_args {
 	};
 };
 
+/*!
+ * \brief Update a channel's existing PBX with new arguments
+ *
+ * \param c channel on which the PBX is running
+ * \param args Options for the PBX
+ *
+ * This updates the PBX arguments on an existing PBX with new arguments
+ * that replace any existing arguments. This is useful if PBX arguments
+ * need to be set or modified after the PBX has already been started.
+ * Arguments can be provided even if none were provided to start the PBX initially.
+ *
+ * If possible, avoid this function and use ast_pbx_run_args instead.
+ *
+ * No channel locks need to be held when calling this function.
```

Comment:
This doesn't really clarify what it is doing. It should state "This function will lock the channel."